### PR TITLE
Translated description in english

### PR DESCRIPTION
**Pull request by mdupierreux**

### CODE REVIEW

File: radar/2023-06-10/Alamofire.md [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
 ---
 
-<p>framework <b>d'appels réseaux</b></p>
-<p><a href="https://github.com/Alamofire/Alamofire">Lien vers git</a> <br /> <br />
-Peut toujours être utilisé dans les projets repris en tma ou d'ancien projet.<br />
-Il serait mieux cependant d'utiliser le language natif pour faire des appels réseaux : <a href="https://developer.apple.com/documentation/foundation/urlrequest">doc Apple</a></p>
+<p>Network calls framework</p>
+<p><a href="https://github.com/Alamofire/Alamofire">Link to Github</a> <br /> <br />
+Can still be used in old projects.<br />
+However, it would be better to use the native language to make network calls: <a href="https://developer.apple.com/documentation/foundation/urlrequest">Apple doc</a></p>
```
Comment (nikkow) [inline radar/2023-06-10/Alamofire.md]: I suggest we use our convictions in the tech-radar. Thus, we might express our position in a stronger way, e.g.

```suggestion
Can still be found in legacy projects.<br />
However, we strongly recommend to use the native language to make network calls: <a href="https://developer.apple.com/documentation/foundation/urlrequest">Apple doc</a></p>
```

File: radar/2023-08-31/GoogleMap for iOS.md [hunk 2]
```diff
@@ -1,11 +1,10 @@
...
+tags: [cartograpy, iOS]
 ---
 
 <p><b>Google Map</b></p>
-<em><a href="https://developers.google.com/maps/documentation/ios-sdk/config?hl=fr">Documentation officiel</a></em>
-<p> Il est préférable d'utiliser les outils natif des OS. Donc google map sera le plus recommandé sur Android mais sur iOS on lui préferera Apple Map
-</p>
\ No newline at end of file
+<em><a href="https://developers.google.com/maps/documentation/ios-sdk/config?hl=fr">Official documentation</a></em>
+<p> It is preferable to use the native OS tools. So Google Map will be the most recommended on Android but on iOS we will prefer Apple Map</p>
```
Comment (nikkow) [inline radar/2023-08-31/GoogleMap for iOS.md]:
```suggestion
<p> It is preferable to use the native OS tools. So Google Map will be the most recommended on Android but on iOS we will prefer the official MapKit SDK.</p>
```

File: radar/2023-08-31/SonarQube.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-tags:       [code quality, Android, iOS, React native]
+title: "SonarQube"
+ring: adopt
+quadrant: tools
+tags: [code quality, Android, iOS, React native]
 ---
 
-<p>outil d' <strong>analyse de code</strong>
+<p>Code analysis tool</p>
 <em><a href="https://sonar.smile.fr/projects/favorite">lien de l'outil</a></em>
```
Comment (nikkow) [inline radar/2023-08-31/SonarQube.md]:
```suggestion
```

File: radar/2023-08-31/SonarQube.md [hunk 4]
```diff
@@ -1,11 +1,11 @@
...
-<p> Permet d'analyser le code sur base d'un linter et de referencer des codes smell, des problèmes de duplication de code ou encore des vulnérabilités
-</p>
\ No newline at end of file
+<p> Allows you to analyze the code based on a linter and reference smell codes, code duplication problems or even vulnerabilities
```
Comment (nikkow) [inline radar/2023-08-31/SonarQube.md:9]: We might also link to the SonarQube Bitrise step written by @ptyjo

File: radar/2023-08-31/SwiftUI.md [hunk 5]
```diff
@@ -1,12 +1,12 @@
...
-<p><b>Language de developpement</b></p>
-<em><a href="https://developer.apple.com/xcode/swiftui/">Documentation Apple</a></em>
-<p>Nouveau language de developpement Apple. Pour le moment le language commence à être stable en iOS 15 minimum.
-en iOS14 beaucoup de point sont à contourner pour un bon fonctionnement et des liens sont a faire avec uikit. iOS 16 apporte également des nouveautés non rétro compatible.
-</p>
\ No newline at end of file
+<p><b>Development language</b></p>
+<em><a href="https://developer.apple.com/xcode/swiftui/">Apple Documentation</a></em>
+<p>New Apple development language. For the moment the language is starting to be stable in iOS 15 minimum.
+in iOS14 many points must be bypassed for proper functioning and links must be made with uikit. iOS 16 also brings new non-retro compatible features.
```
Comment (nikkow) [inline radar/2023-08-31/SwiftUI.md]:
```suggestion
<p>Apple's officiel development language. We consider this language to be usable and stable from iOS 15 onwards.
On iOS 14, many aspects must be bypassed or worked around for proper functioning and links must be made with UIKit. 
iOS 16 also brings new non-retro compatible features.
```

**Review by nikkow — CHANGES_REQUESTED** [review]

File: radar/2023-06-10/Alamofire.md (hunk 1 above)
Comment (mdupierreux) [reply radar/2023-06-10/Alamofire.md]: Since it's an iOS tech I'll let @NeopixlAnthony (or someone else) complete this part

**Review by nikkow — APPROVED** [review]

**Review by NeopixlAnthony — APPROVED** [review]
ok

**Pull request merged**